### PR TITLE
Ply output

### PR DESCRIPTION
**Pull request by rhohndorf**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -102,44 +102,26 @@ Validators update the miner every 20 minutes. It's normal not to observe request
 nc -vz [Your Miner IP] [Port]
 ```
```
Comment (the-mx) [inline README.md:104]: I'm against having different way of running the node for miner and validator. Either both should be with pm2 or make. And with make we need explicitly write that pm2 is used underneath. Plus, we need to say what file should be edited.

File: neurons/common/synapses.py [hunk 2]
```diff
@@ -0,0 +1,44 @@
+from typing import Literal
```
Comment (the-mx) [inline neurons/common/synapses.py:1]: Please, remove this file

File: neurons/old/version.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+__version__ = "0.0.1"
+version_split = __version__.split(".")
```
Comment (the-mx) [inline neurons/old/version.py:2]: Please, remove this file

File: neurons/common/synapses.py (hunk 2 above)
Comment (rhohndorf) [reply neurons/common/synapses.py:1]: done

File: neurons/old/version.py (hunk 3 above)
Comment (rhohndorf) [reply neurons/old/version.py:2]: Should I remove the whole folder then ? It's empty.

File: README.md [hunk 4]
```diff
@@ -102,44 +102,27 @@ Validators update the miner every 20 minutes. It's normal not to observe request
...
 
-## Validator
-
-To operate the validator, the validator neuron and validation endpoints must be initiated. 
-
-### Validation Endpoint
+## Validation
```
Comment (the-mx) [inline README.md:105]: Validator ?

**Review by the-mx — APPROVED** [review]

**Pull request merged**